docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -160,7 +160,7 @@ Now you can start all services:
 
 .. code-block:: bash
 
-    docker-compose up
+    docker compose up
```

Comment:
this is not a mistake
see https://github.com/apache/airflow/pull/27792 and https://github.com/apache/airflow/pull/28194

I will raise a PR to clarify that this is not a mistake as this is the 3rd time people are asking to fix it :)

Reply:
Sorry, I just wanted to help. But it is still unclear to me.
As I understood in [#28194](https://github.com/apache/airflow/pull/28194#issuecomment-1341479006), Airflow is using docker compose v2.
![image](https://user-images.githubusercontent.com/34426867/210561025-a5f36184-c13c-4bf0-b926-d244218a5e7a.png)

![image](https://user-images.githubusercontent.com/34426867/210561070-54053caf-10c9-4dab-a551-f669172094bc.png)

Reply:
Current Page
![image](https://user-images.githubusercontent.com/34426867/210562619-6b24cdcd-f0db-4efa-9b9e-f9097246e871.png)